src/base.py

```diff
@@ -8,6 +10,8 @@
 
 
 class ParsedHTML(BaseModel):
```

Comment:
it's a shame we can't have a shared base model for "parsed data"

Reply:
We probably could - but in separate repos where would be put the base model?